src/app/dashboard/import-export/export-modal/export-modal.component.ts

```diff
@@ -21,6 +21,7 @@ export class ExportModalComponent implements OnInit {
   canExport: boolean = false;
   noDirectoryAssessments: Array<ImportExportAssessment>;
   exportName: string;
+  directory: Directory;
```

Comment:
I don't think directory needs to be a class property - can keep original "let directory.."